war/pom.xml

```diff
@@ -82,6 +81,10 @@ THE SOFTWARE.
           <artifactId>metainf-services</artifactId>
           <groupId>org.kohsuke.metainf-services</groupId>
         </exclusion>
+        <exclusion>
```

Comment:
FYI this broke JCasC, I've created a PR to fix it: https://github.com/jenkinsci/configuration-as-code-plugin/pull/1359